src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/NuGetPackageMetadata.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public record NuGetPackageMetadata
+    {
+        public string? OwnerId { get; init; }
```

Comment:
nit: please add comment explaining what this data looks like. is it an email address? a list of github usernames?